addons/mail/static/src/new/chat/chat_window_container.js

```diff
@@ -11,9 +11,27 @@ export class ChatWindowContainer extends Component {
 
     setup() {
         this.messaging = useMessaging();
+        this.state = useState({
+            isActive: false,
+        });
```

Comment:
I think you won't need this local state anymore with awaiting `this.messaging.isReady`. You can keep relying on observing `this.messaging.state.discuss.isActive`, and let re-render happen from that.